cafebabel/templates/articles/to-translate.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 {% import "articles/macros.html" as macros %}
+{% from "articles/macros.html" import card %}
```

Comment:
Macros are already imported, you can reuse `macros.card()` directly.

Reply:
I think I had an error without that. I wouldn't have thought about import it again otherwise. Will have a look closer on Monday

Reply:
oui voilà, I get `jinja2.exceptions.UndefinedError: 'card' is undefined` otherwise. I guess `import` was not the right way.

Reply:
you need to prefix `card` like that: `macros.card`, did you do that?

Reply:
https://github.com/cafebabel/cafebabel.com/commit/7287ca0ebba14fb2da5410c6ba19cdc3f6937090 isn't it fine this way? with `{% from "articles/macros.html" import card %}` ?

Reply:
@cafebabel-webmaster if you only use that macro that's the way to go 👍

Reply:
:-)